Gruntfile.js

```diff
@@ -80,7 +80,7 @@ module.exports = function (grunt) {
      * Compiles a production build of CyberChef into a single, portable web page.
      */
     function runInliner() {
-        var inlinerError = false;
```

Comment:
this var broke, without the callbacks working properly, so might as well fix them at the same time.